web/courses.go

```diff
@@ -292,7 +292,6 @@ func (s *QuickFeedService) updateSubmissions(request *qf.UpdateSubmissionsReques
 	}, false); err != nil {
```

Comment:
@0xf8f8ff and @JosteinLindhom: The `getAssignmentWithCourse()` call above seems to be here only to check that the course and assignment exist. Is this necessary? 

The `UpdateSubmissions()` database call (below) also checks via the `Where` clause that the assignment exists.
```go
return db.conn.
	Model(query).
	Where("assignment_id = ?", query.AssignmentID).
	Where("score >= ?", query.Score).
```

Reply:
Looks absolutely unnecessary to me. I think it should be ok to just remove this method.

Reply:
Agreed